index.html

```diff
@@ -38,5 +38,169 @@ <h1 class="lines"> Hey there <br>
+        <div class="text-aline">
+          </div>
+          <div class="text-about">
+            <div class="plane-container">
+              <h3>About me</h3>
+              <div class="plane-image">
+                <img src="images/plane.svg" alt="plane">
+              </div>
+            </div>
+            <div class="test2">
+              <div>
+                <p class="h6"> I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                  don’t hesitate to contact me.</p>
+              </div> 
+            </div>
+              <div class="button2">
+                <button class="button-resume" type="button">Get my resume </button>
+              </div>
+          </div>
+          <div class="plane">
+            <img src="images/square.svg" alt="square">
+          </div>
+        </div>
+         <div class="section6">
+          <article class="qoute">
+            <div class="holder-text">
+              <img src="images/holder.svg" alt="holder">
+              <div class="holder-text1">
+                <img src="images/ellipse.svg" alt="holder">
+               <div class="Language">
+                <p>Languages</p>
+                </div>
+                  <ul class="media">
+                    <li>JavaScript</li>
+                    <li>Ruby</li>
+                    <li>HTML</li>
+                    <li>CSS</li>
+                  </ul>
+              </div>
+            </div>
+            </article>
+              <article class="qoute">
+                  <div class="holder-text">
+                  <img src="images/holder.svg" alt="holder">
+                  <div class="holder-text1">
+                    <img src="images/ellipse2.svg" alt="holder">
+                    <div class="Language">
+                      <p>Frameworks</p>
+                    </div>
+                    <ul class="media">
+                      <li>React.js</li>
+                      <li>Ruby on Rails</li>
+                      <li>RSpec</li>
+                      <li>Capybara</li>
+                      <li>Selenium</li>
+                    </ul>
+                    </div>
+                 </div>
+                  </article>
+                  <article class="qoute">
+                    <div class="holder-text">
+                    <img src="images/holder.svg" alt="holder">
+                    <div class="holder-text1">
+                      <img src="images/skills.svg" alt="holder">
+                      <div class="Language">
+                        <p>Skills</p>
+                      </div>
+                      <ul class="media">
+                        <li>Database Management</li>
+                        <li>Version Controle</li>
+                        <li>CLI</li>
+                        <li>WEb Development</li>
+                        <li>API Design</li>
+                      <ul>
```

Comment:
- [ ] As mentioned by the previous reviewer, you should have:
- Your technology categories ("Languages", "Frameworks" and "Skills") in a `ul`, & each of them in a `li`. 
- The technologies in each category(e.g., "JavasScript", "HTML" and "CSS") should also be in a `ul` and each of them in a `li`